Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -18,14 +18,63 @@ import ArcGIS
 
 class ARExample: UIViewController {
     
-    let arView = ArcGISARView(renderVideoFeed: true, tryUsingARKit: true)
+    typealias sceneInitFunction = () -> AGSScene
+    
+    /// The scene creation functions plus labels.  The functions create a new scene and perform any necessary `ArcGISARView` initialization.  This allows for changing the scene and AR "mode" (table top or full-scale).
+    private var sceneInfo: [(sceneFunction: sceneInitFunction, label: String)] = []
+    
+    /// The current scene info.
+    private var currentSceneInfo: (sceneFunction: sceneInitFunction, label: String)? {
+        didSet {
+            guard let label = currentSceneInfo?.label else { return }
+            statusViewController?.currentScene = label
+        }
+    }
+    
+    /// The `ArcGISARView` that displays the camera feed and handles ARKit functionality.
+    private let arView = ArcGISARView(renderVideoFeed: true, tryUsingARKit: true)
+    
+    /// Denotes whether we've performed a hit test yet.
+    private var didHitTest: Bool = false
+
+    // View controller displaying current status of `ARExample`.
+    private let statusViewController: ARStatusViewController? = {
+        let storyBoard = UIStoryboard(name: "ARStatusViewController", bundle: nil)
+        let vc = storyBoard.instantiateInitialViewController() as? ARStatusViewController
+        return vc
+    }()
+    
+    /// Used when calculating framerate.
+    private var lastUpdateTime: TimeInterval = 0
+    
+    /// Overlay used to display user-placed graphics.
+    private let graphicsOverlay: AGSGraphicsOverlay = {
+        let overlay = AGSGraphicsOverlay()
+        let properties = AGSLayerSceneProperties(surfacePlacement: .relative)
+        return overlay
+    }()
+    
+    /// The observer for the `SceneView`'s `translationFactor` property
+    private var translationFactorObservation: NSKeyValueObservation?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         
         // Set ourself as delegate so we can get ARSCNViewDelegate method calls.
         arView.arSCNViewDelegate = self
         
+        // Set ourself as touch delegate so we can get touch events.
+        arView.sceneView.touchDelegate = self
+        
+        // Add our graphics overlay to the sceneView.
+        arView.sceneView.graphicsOverlays.add(graphicsOverlay)
+        
+        // Observe the `cameraController.translationFactor` property and update status when it changes.
+        translationFactorObservation = arView.observe(\ArcGISARView.translationFactor, options: [.initial, .new]){ [weak self] arView, change in
```

Comment:
I want to be notified of the initial value so I can set the translationFactor property in the StatusViewController.  Without using `.initial`, that doesn't happen.

Reply:
Yes. That's what passing the `initial` option gives you, which I included in my suggested change. The `new` option causes the `newValue` property of `change` to be populated. Unless you're reading that new value, passing the `new` option just caused extra work to be done for no reason. Does that make sense?

Reply:
Will just passing `.initial` still notify me of changes to the value?

Reply:
Yes. You'll be notified of changes to the value no matter what, but specifying `initial` also means that you'll be notified right away:

> If specified, a notification should be sent to the observer immediately, before the observer registration method even returns.

If passing `initial` meant that you only got the initial value, it would be much simpler to just read the value directly.